Libraries/LibGUI/TextEditor.cpp

```diff
@@ -301,7 +301,7 @@ void TextEditor::mouseup_event(MouseEvent& event)
 void TextEditor::mousemove_event(MouseEvent& event)
 {
     m_last_mousemove_position = event.position();
-    if (m_in_drag_select) {
+    if (m_in_drag_select && !m_in_drag_select) {
```

Comment:
That would just be `if (false) {`, and it would cause the scrolling to stop if the timer is disabled (e.g. when you leave then reenter the screen).
Maybe change it `m_in_drag_select && !m_automatic_selection_scroll_timer.is_active()` instead ?

Reply:
Wow. Evidently pushing a quick PR at 2AM on a weekday morning results in stuff like this.

That being said, stopping the scroll when re-entering the widget is expected behaviour, is it not?
I pushed a fix just now. Thanks for catching this!